transport.go

```diff
@@ -166,6 +166,17 @@ func getTransport(opts TransportOptions, encoding encoding.Encoding, tracer open
 		return transport.NewTChannel(topts)
 	}
 
+	if protocol == "grpc" {
```

Comment:
no coverage here, can we please add tests for this

Reply:
Testing added.